src/app/collective-rewards/components/BuildersCallToAction/BuildersCallToAction.tsx

```diff
@@ -68,32 +70,39 @@ interface BuildersCallToActionProps {
   className?: string
 }
 
-export const BuildersCallToAction: FC<BuildersCallToActionProps> = ({
-  rifRewards,
-  rbtcRewards,
-  className,
-}) => {
+export const BuildersCallToAction: FC<BuildersCallToActionProps> = ({ rifRewards, rbtcRewards }) => {
+  const collapsibleContent = (
+    <Paragraph className="text-v3-text-0 order-2 pb-6 px-0 md:px-2">
+      Join the Collective as a Builder and earn for delivering impact. Be part of Bitcoin&apos;s most aligned
+      innovation network.
+    </Paragraph>
+  )
+
   return (
-    <CallToActionCard
-      title={<BuildersTitle />}
-      banner={<BuildersBanner />}
-      className={cn('bg-v3-text-80 w-full', className)}
-    >
-      <MetricsContainer className="px-6 pb-10 pt-0 bg-v3-text-80 items-start divide-y-0">
+    <CallToActionCard className="bg-v3-text-80 rounded-sm p-4" defaultOpen={true}>
```

Comment:
By default it should be closed
```suggestion
    <CallToActionCard className="bg-v3-text-80 rounded-sm p-4" defaultOpen={false}>
```